Sending and Receiving Coins Using your Skywallet.md

```diff
@@ -0,0 +1,100 @@
+# Sending and Receiving Coins Using your Skywallet
+
+Sending and receiving of Coins are two of the most basic features of your Skywallet.
+
+To send or receive Coins using your Skywallet, you have to initialize and personalize your Skywallet, details on how to initialize and personalize your Skywallet is explained in the following sections :
+
+**[Initialization of your Skywallet](https://github.com/skycoin/hardware-wallet/wiki/Initialize-the-wallet-setting-up)**
+
+**[Personalization of your Skywallet](https://github.com/skycoin/hardware-wallet/wiki/Getting-to-know-the-wallet)**
+
+Once you have completed your initialization and personalization of your Skywallet using your Desktop Wallet you can see your Skywallet in the list of Wallets configured using your Desktop Wallet.
+
+<screenshot of the Desktop Wallet showing the list of configured Wallets>
+
+## Receive Coins
+
+To receive Coins to your Skywallet, you need the address of your Skywallet.
+
+To get the address of the Skywallet, follow these steps :
+
+**Step 1** - Connect your Skywallet to your computer via USB.
+
+**Step 2** - Open the Desktop Wallet from your computer.
+
+**Step 3** - The window will display the list of all the Wallets configured using your Desktop Wallet, from this list you can click on your Skywallet name to get the Wallet address of your Skywallet.
+
+However, only the partial address will be visible for you.
+
+<screenshot of the Desktop Wallet after clicking on the Skywallet name and displaying the partial address>
+
+**Step 4** - Click on the **Press to Show** button.
+
+Once you have clicked on the "Press to Show" button, the address of the Skywallet will be displayed both by the Desktop Wallet as well as your Skywallet.
+
+**Wallets Showing Same Addresses**
+
+<screenshot and image of the same address being displayed>
+
+**NOTE - Please check whether both these addresses are same. If it is not the same, then cancel the operation and stop using both the Skywallet and Desktop Wallet. Immediately contact Support by visiting this link - [store.skycoin.net/pages/support](store.skycoin.net/pages/support), or you can send an email detailing the issue to **support@skycoin.zendesk.com**.  
+It is imperative you do this as showing different addresses means your Wallets might have been compromised and using the same would lead you to the loss of your Coins.**
+
+**Wallets Showing Different Addresses**
+
+<screenshot and image of different addresses being displayed>
+
+**Step 5** - After confirming both the addresses are same, proceed to confirm the operation on your Skywallet. 
```

Comment:
the same

Reply:
Confirm what? Isnt receiving possible without any confirmation?

Reply:
> Confirm what? Isnt receiving possible without any confirmation?

This was mentioned in the existing wiki, have not personally tried it.

Reply:
Checked this, there is no confirmation message as such, just a dialogue box appearing that says, you can view the address by clicking on Show address button. Hence, will remove the second part of this sentence.